providers/implementations/rands/drbg.c

```diff
@@ -922,3 +923,31 @@ int ossl_drbg_set_ctx_params(PROV_DRBG *drbg, const OSSL_PARAM params[])
         return 0;
     return 1;
 }
+
+/* Confirm digest is allowed to be used with a DRBG */
+int ossl_drbg_verify_digest(ossl_unused OSSL_LIB_CTX *libctx, const EVP_MD *md)
```

Comment:
Note: That I used securitycheck.c + securitycheck_default.c and securitycheck_fips.c for the other options..

Reply:
Suggesting that I put this one there too?

It's a pretty silly requirement and the more isolated the better.

Reply:
I was trying to isolate the code to one place. It doesnt matter I guess.